Rubberduck.RegexAssistant/Atoms/CharacterClass.cs

```diff
@@ -59,20 +61,14 @@ private IList<string> ExtractCharacterSpecifiers(string characterClass, bool spe
                     }
                 }
 
-                if (spellOutWhitespace 
-                    && WhitespaceToString.IsFullySpellingOutApplicable(specifier.Value, out var spelledOutWhiteSpace))
-                {
-                    result.Add(spelledOutWhiteSpace);
-                }
-                else
-                {
-                    result.Add(specifier.Value);
-                }
+                result.Add(spellOutWhitespace && WhitespaceToString.IsFullySpellingOutApplicable(specifier.Value, out var spelledOutWhiteSpace)
+                    ? spelledOutWhiteSpace
+                    : specifier.Value);
             }
             return result;
         }
 
-        public string Description => string.Format(InverseMatching 
+        public string Description(bool notApplicable) => string.Format(InverseMatching 
```

Comment:
FWIW I don't like the changing of the parameter name. That makes for a confusing interface. Even though the classes right now don't use the feature, there is no reason they won't in future. I'd just have them all as be `Description(bool spellOutWhitespace)` and have those classes just not use it. 

Come to think of it, why there's no `IDescribable` interface?

Reply:
[IDescribable](https://github.com/IvenBach/Rubberduck/blob/Issue2776_RegexAssistantWhitespace/Rubberduck.RegexAssistant/IDescribable.cs) does exist.